src/lv_objx/lv_img.c

```diff
@@ -355,8 +355,8 @@ static bool lv_img_design(lv_obj_t * img, const lv_area_t * mask, lv_design_mode
 
         if(ext->cf == LV_IMG_CF_TRUE_COLOR || ext->cf == LV_IMG_CF_RAW) cover = lv_area_is_in(mask, &img->coords);
 
-        const lv_style_t * style = lv_img_get_style(img, LV_IMG_STYLE_MAIN);
```

Comment:
Why was this rename necessary?

Reply:
`style` was already declared in the enclosing scope: https://github.com/no1wudi/lvgl/blob/0e2482cdd1cffa53b33b5e87526e5d20daa9e5e0/src/lv_objx/lv_img.c#L349

Reply:
Oh god... I was looking for directly this, but haven't noticed that declaration. 
Thanks!